### ISSUE TITLE
Preview Images Aggressively Cropped

### ISSUE THREAD

**Issue by BoffinBrain** [userstyles-world/userstyles.world#330]
I've just started migrating over from UserStyles.org so I figured I'd report this as I go.

On both the regular and the 'redesigned' style view page, the preview image container has a hard-coded height. If the preview image or screenshot doesn't exactly match the aspect ratio of the container, it's cropped at the top and bottom, with no way to see the rest:

![Screenshot 2024-12-18 at 00-17-42 MixCloud Accessibility Fixes — UserStyles world](https://github.com/user-attachments/assets/50548110-541f-4340-969d-531e6965cd8e)

This can be fixed by removing `align-items: center;` from `.card .screenshot picture` in the CSS. Given that the screenshot container includes a blurred version, I assume this was the intended way to display it:

![Screenshot 2024-12-18 at 00-18-03 MixCloud Accessibility Fixes — UserStyles world](https://github.com/user-attachments/assets/f1b249de-5d0d-4b27-9693-e7bcbbf9c334)

**Browser**: Firefox 134 on Windows

**Comment by BoffinBrain** [comment]
Yes, I saw that the USw listing previews are similarly cropped, although with those it aligns them at the top, cropping off the bottom of the picture. This makes more sense. Still worth doing the one-line change though, IMO. Not everyone will be aware of the cropping or know how to optimize their screenshots.

**Comment by Athari** [comment]
Huh, somehow it never occurred to me that I'm supposed to set the aspect ratio of my screenshots to 16:9 instead of fixing it on the client side (with [my userstyle](https://userstyles.world/style/20883/userstyles-world-wide-compact-ath)). After all, most users do see the screenshots from Stylus's popup...

@astyled 
I think the hint to use 16:9 images should be put on the Add page. It's rather important, if you have no plans to change the website's behavior.
